wandb/sdk/launch/_project_spec.py

```diff
@@ -39,6 +41,14 @@ class LaunchSource(enum.IntEnum):
     JOB: int = 5
 
 
+class EntrypointDefaults(List[str]):
+    PYTHON = ["python", "main.py"]
+
+
+class LaunchType(str):
```

Comment:
This seems kinda intense, might be simpler and easier to just have a variable `JOB_LAUCH_TYPE` rather than this class that has a single property

Reply:
I guess this only makes sense if we think there will ever be a need/want to support other default entry points. Like if the container is a bash operator or something and we want to have a `['sh', 'main.sh']` or something. If not then yeah I totally agree this is verbose for no reason.

Reply:
Where this is being used I think this is unnecessary. We can just write "launch_job", if I understand right the created Job should be entrypoint agnostic right?